### PR TITLE
Prompt experiments table

### PR DESCRIPTION
**Pull request by chitalian**

### CODE REVIEW

File: supabase/migrations/20240412000067_prompts-experiments.sql [hunk 1]
```diff
@@ -0,0 +1,125 @@
+CREATE TABLE "public"."prompt_v2" (
```
Comment (colegottdank) [inline supabase/migrations/20240412000067_prompts-experiments.sql:1]: soft delete on prompt_v2?

File: supabase/migrations/20240412000067_prompts-experiments.sql [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+    model TEXT,
+    organization UUID NOT NULL,
+    CONSTRAINT fk_prompt FOREIGN KEY (prompt_v2) REFERENCES prompt_v2(id),
+    CONSTRAINT fk_organization FOREIGN KEY (organization) REFERENCES organization(id),
+    CONSTRAINT check_major_version CHECK (major_version >= 0),
+    CONSTRAINT check_minor_version CHECK (minor_version >= 0)
+);
+CREATE INDEX idx_prompts_versions_major ON prompts_versions(organization, major_version DESC);
+CREATE INDEX idx_prompts_versions_minor ON prompts_versions(organization, major_version DESC, minor_version DESC);
+CREATE UNIQUE INDEX idx_prompts_versions_model ON prompts_versions(organization, prompt_v2, major_version DESC, minor_version DESC);
```
Comment (colegottdank) [inline supabase/migrations/20240412000067_prompts-experiments.sql:32]: unique index includes DESC for the versions?

File: supabase/migrations/20240412000067_prompts-experiments.sql (hunk 1 above)
Comment (chitalian) [reply supabase/migrations/20240412000067_prompts-experiments.sql:1]: good point thanks

File: supabase/migrations/20240412000067_prompts-experiments.sql (hunk 2 above)
Comment (chitalian) [reply supabase/migrations/20240412000067_prompts-experiments.sql:32]: I guess it doesn't really matter, I will kill it.

**Pull request merged**